src/array_util.rb

```diff
@@ -0,0 +1,43 @@
+class ArrayUtil
+
+  def self.sum_even_numbers(numbers)
+    numbers&.select(&:even?)&.sum
```

Comment:
Why use the safe operator `&`?

Is there a better alternative?

Reply:
I use safe operator to return ```nil``` if method is called with ```nil``` parameter.
I miss something?

Reply:
Update:
I don't need to use safe operator or to check for ```nil``` because the problem says I have an array.
I thought I had to check for ```nil``` to be sure.